### ISSUE TITLE
Aria label in footer feedback link is incorrect if the feedback_title arg isn't passed in

### ISSUE THREAD

**Issue by KPobeeNorris** [citizensadvice/design-system#3508]
## Summary of problem

We've recently been made aware of an issue with the aria-label in the footer feedback link being malformed if the feedback_title arg isn't passed through to the design system.  If no feedback_title is passed in then the aria label is set to ` (opens in new tab)`.  

In the [footer html file](https://github.com/citizensadvice/design-system/blob/main/engine/app/components/citizens_advice_components/footer.html.erb#L8) there's an `or` statement that falls back to the default feedback title if the title is missing.  But this behaviour isn't extended to/replicated in the aria-label in the [ruby file](https://github.com/citizensadvice/design-system/blob/main/engine/app/components/citizens_advice_components/footer.rb#L67), it just uses `title`, so it will be nil if the title hasn't been passed in.

## Steps to replicate

Render the footer component without the feedback_title argument e.g. 

`= render FooterComponent.new(current_path: request.path, 
                                                         feedback_survey_id: "/some-feedback-url/", 
                                                         columns: advisernet_footer_nav_links)`

## Expected behaviour

The aria-label falls back to use the same feedback text as the title; if feedback_title is nil

## Actual behavior

The aria-label is set to ` (opens in a new tab)` if feedback_title isn't passed through

## Context

- url
We've updated public-website to pass the feedback_title everywhere that we use the Footer component so there's no url that I know of where this bug is now present

- screenshot
The feedback link component when rendered:
![Screenshot 2024-10-15 at 08 20 56](https://github.com/user-attachments/assets/4ace9e9a-4071-4540-891d-1eae2a97b560)

This is a screenshot of the aria-label before the update to public-website:
![Screenshot 2024-10-15 at 08 21 23](https://github.com/user-attachments/assets/a04a63fd-8a52-4ac9-94d9-d141fef21251)

- browser / os / device
All/any
